cmd/metricshub/main.go

```diff
@@ -92,6 +95,13 @@ func makeHandler(store store, c *config) http.HandlerFunc {
 			return
 		}
 
+		ip, err := extractClientIP(r)
+		if err != nil {
+			log.Warn().Msg("could not get ip address")
+		}
+
+		log.Debug().Str("node_id", req.NodeID).Str("ip_address", ip).Msg("got metric from node")
```

Comment:
Added this to see if we are getting the same node id from different IP address in the metrics hub server